test/test_masked.py

```diff
@@ -162,6 +162,22 @@ def apply_masked_normalization_along_dim(op, input, *args, **kwargs):
 
 masked_ops = [op for op in op_db if op.name.startswith('_masked.')]
 masked_ops_with_references = [op for op in masked_ops if op.name.rsplit('.', 1)[-1] in reference_functions]
+masked_ops_with_mixed_layouts_support = [op for op in masked_ops if op.supports_sparse or op.supports_sparse_csr]
+
+
+def to_strided(obj):
+    """Convert the tensor content of object to strided tensor content.
+    """
+    if isinstance(obj, torch.Tensor):
```

Comment:
nit: You could also use [torch.is_tensor](https://pytorch.org/docs/master/generated/torch.is_tensor.html#torch.is_tensor).

Reply:
Fixed.